website/src/pages/Blog/styles.ts

```diff
@@ -151,7 +151,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     flexGrow: 1,
   },
   cardSize: {
-    maxWidth: "480px",
+    maxWidth: "49%",
```

Comment:
Here, max-width value is correct. If you see the screenshot below, grid CSS is overriding the custom CSS. If you use `!important` with the existing value, it will fix the issue

<img width="1440" alt="Screenshot 2021-07-09 at 10 44 01 AM" src="https://user-images.githubusercontent.com/34312966/125026721-cd61e200-e0a2-11eb-939c-1b0c94469d84.png">